src/core/core.scss

```diff
@@ -0,0 +1,3 @@
+main {
```

Comment:
The component Core was just my choice, the code in **Core** can be moved into the root component, but since we subscribe to the state there, I preferred to separate them (currently). _I like to use child component as much as possible_.

On the src level, I like to have the primary features, I wouldn't make `Auth Guard` to be a part of **SharedModule**, because I feel shared components should have secondary features, but this is all opinion choices, it doesn't mean this approach is better than the other.

Keeping in mind that I will be re organizing the folder structure from time to time as we add more features.